source/include/FreeRTOSIPConfigDefaults.h

```diff
@@ -1161,4 +1161,32 @@
     #define ipconfigRA_IP_TEST_TIME_OUT_MSEC    ( 1500U )
 #endif
 
+#ifndef ipconfigEVENT_QUEUES
+    #define ipconfigEVENT_QUEUES    1
+#endif
+
+#ifndef ipconfigPACKET_PRIORITIES
+    #if ipconfigEVENT_QUEUES > 1
+        #define ipconfigPACKET_PRIORITIES    8
+    #else
+        #define ipconfigPACKET_PRIORITIES    1
+    #endif
+#endif
+
+#ifndef ipconfigPACKET_PRIORITY_MAPPING
+    #if ipconfigPACKET_PRIORITIES == 8 && ipconfigEVENT_QUEUES == 3
+        #define ipconfigPACKET_PRIORITY_MAPPING    { 0, 1, 1, 1, 2, 2, 2, 2, }
```

Comment:
Just trying to understand this scheme here. Please do correct me if I am wrong.

All messages with priority 0 will be added to Queue 0.
And all messages with priority 1-3 will be added to Queue 1.
And all messages with priority 4-7 will be added to Queue 2.

Is that right?
If so, then what is the point of having 8 different priorites if you just have 3 queues?

Reply:
The idea is based on the current linux implementation, where tc-prio has 3 fifos. The 8 comes from the size of the vlan pcp which has 3 bits so 8 priorities. I do hw mappings inside of the driver and add vlan tags with the priority.

Reply:
Could you please add a bit of comment about the solution here? It is confusing to see 3 queues, and 8 different priorities.

Reply:
Add added a big comment below. I will add comment in the code as well.

Reply:
Wouldn't it be better to have the number of priorities and the number queues to be equal in the default configuration (if we are keeping a default mapping), to avoid users getting confused about packets with different priorites getting into the same queue (depending on the mapping) and processed together regardless of their actual priorities.